### PR TITLE
create-file-structure

### PR DESCRIPTION
**Pull request by Rawanzaqout**

### CODE REVIEW

**Review by HebaAhmad1 — DISMISSED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

File: covid vaccine tracker/covid vaccine tracker.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./covid vaccine tracker.css">
```
Comment (Ibrahim-Jarada) [inline covid vaccine tracker/covid vaccine tracker.html:7]: don't use spaces in your href or src so change the file name

check this link [https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files#an_aside_on_casing_and_spacing](url)

**Review by Ibrahim-Jarada — CHANGES_REQUESTED** [review]

**Review by AbdallahGot — APPROVED** [review]

**Review by AbdallahGot — CHANGES_REQUESTED** [review]
now you have two folders for the same page🤔, so you should delete the useless folder (the folder called covid vaccine tracker)

**Review by Ibrahim-Jarada — DISMISSED** [review]

**Review by AbdallahGot — DISMISSED** [review]

**Review by NadaSaleh20 — DISMISSED** [review]

File: advice/advice.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./advice.css">
+    <title>Document</title>
```
Comment (lina-jamal) [inline advice/advice.html:8]: add a descriptive title for every page.

**Review by lina-jamal — COMMENTED** [review]
please add a description in pull request to show us what you do, and link the pull request with the related issue.
you shouldn't add the  number of issue in pull request title as ``` 2-create-file-structure  ``` just title without number

**Review by HebaAhmad1 — APPROVED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

**Review by MontherIsmail — APPROVED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Pull request merged**